frontera/contrib/scrapy/converters.py

```diff
@@ -98,8 +99,8 @@ def from_frontier(self, response):
 
 
 def _find_method(obj, func):
-    if obj and hasattr(func, 'im_self') and func.im_self is obj:
-        return func.im_func.__name__
+    if obj and hasattr(func, '__self__') and func.__self__ is obj:
```

Comment:
That's fine. Is there tests for serializing/unserializing requests with callback function?